features/step_definitions/finance/settings_steps.rb

```diff
@@ -19,7 +19,6 @@
 
 Then "they should not be able to review the charging and gateway billing settings" do
   visit admin_finance_settings_path
-  assert_equal 403, status_code
```

Comment:
this stopped working somehow?

Reply:
Yes, `status_code` is not meant for browser tests so it's not available anymore.